projects/plugins/jetpack/extensions/shared/currencies.js

```diff
@@ -1,10 +1,120 @@
-import { CURRENCIES } from '@automattic/format-currency';
-
 // Removes all dots (`.`) from the end of a string.
 function removeTrailingDots( string ) {
 	return String( string || '' ).replace( /\.+$/, '' );
 }
 
+/**
+ * Currency data for formatting currencies.
+ * This is an internalized version of the CURRENCIES object previously imported from format-currency.
```

Comment:
Can you add a note that reads something like:

```
IMPORTANT: currency formatting is dependent on locale, not on currency code, so using `grouping` and `decimal` here will provide inaccurate results. Please use `@automattic/number-formatters` package for formatting.
```

Reply:
Thanks @sirbrillig ! Certainly. I need to rebase here, and it might be the case we no longer need this variable if https://github.com/Automattic/jetpack/pull/43418 suffices to migrate the donations block straight into using `@automattic/number-formatters` (without the separate currency symbol rendering). I still need to test that, so will return here once done.